.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+python:
+  - 2.6
+  - 2.7
+addons:
+  apt:
+    packages:
+      - tcl8.5
+install:
+    # install easybuild-framework (and dependencies)
+    # use framework 'develop' branch, except when testing 'master'
+    - if [ "x$TRAVIS_BRANCH" = 'xmaster' ]; then FRAMEWORK_BRANCH=master; else FRAMEWORK_BRANCH=develop; fi
+    - easy_install https://github.com/hpcugent/easybuild-framework/archive/${FRAMEWORK_BRANCH}.tar.gz
```

Comment:
OK, seems like it doesn't. I'll figure out the `git` (or `curl`) command we can use to determine the commit ID that corresponds to `develop` at that time.